lib/core/presentation/pages/event/event_control_panel_page/sub_pages/event_approval_setting_page/sub_pages/event_join_request_application_page/event_join_request_application_page.dart

```diff
@@ -0,0 +1,189 @@
+import 'package:app/core/domain/event/entities/event.dart';
+import 'package:app/core/domain/event/entities/event_join_request.dart';
+import 'package:app/core/domain/onboarding/onboarding_inputs.dart';
+import 'package:app/core/domain/user/entities/user.dart';
+import 'package:app/core/domain/user/user_repository.dart';
+import 'package:app/core/presentation/pages/event/event_control_panel_page/sub_pages/event_approval_setting_page/sub_pages/event_join_request_application_page/widgets/event_join_request_application_form.dart';
+import 'package:app/core/presentation/pages/event/event_control_panel_page/sub_pages/event_approval_setting_page/sub_pages/event_join_request_application_page/widgets/event_join_request_application_user_card.dart';
+import 'package:app/core/presentation/widgets/common/appbar/lemon_appbar_widget.dart';
+import 'package:app/core/presentation/widgets/common/button/linear_gradient_button_widget.dart';
+import 'package:app/core/presentation/widgets/theme_svg_icon_widget.dart';
+import 'package:app/gen/assets.gen.dart';
+import 'package:app/gen/fonts.gen.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:app/injection/register_module.dart';
+import 'package:app/theme/color.dart';
+import 'package:app/theme/sizing.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:flutter/material.dart';
+
+class EventJoinRequestApplicationPage extends StatelessWidget {
+  final EventJoinRequest eventJoinRequest;
+  final Event? event;
+  final Function()? onPressApprove;
+  final Function()? onPressDecline;
+
+  const EventJoinRequestApplicationPage({
+    super.key,
+    required this.eventJoinRequest,
+    this.event,
+    this.onPressApprove,
+    this.onPressDecline,
+  });
+
+  bool get isPending =>
+      eventJoinRequest.approvedBy == null &&
+      eventJoinRequest.declinedBy == null;
+
+  @override
+  Widget build(BuildContext context) {
+    final t = Translations.of(context);
+    final colorScheme = Theme.of(context).colorScheme;
+
+    return Scaffold(
+      backgroundColor: LemonColor.atomicBlack,
+      appBar: LemonAppBar(
+        backgroundColor: LemonColor.atomicBlack,
+        title: t.event.eventApproval.application,
+      ),
+      body: FutureBuilder(
+        future: getIt<UserRepository>().getUserProfile(
+          GetProfileInput(
+            id: eventJoinRequest.user ?? '',
+          ),
+        ),
+        builder: (context, snapshot) {
+          User? user = snapshot.data?.fold(
+            (l) => null,
+            (user) => user,
+          );
+
```

Comment:
Should we return error UI if user is null

Reply:
if the user is null then it does only not display the name , so I think it's ok to leave it there, throwing a error UI will not look good